app/lint-baseline.xml

```diff
@@ -1,1090 +1,1112 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
😬 everything about this change is a bummer

Reply:
I'll start chipping away at the baseline next so we see far less of this nonsense, I promise 😁